Add JWT support (#85)
spiffe/spiffe-helper: README.md

 |`cert_dir`                | Directory name to store the fetched certificates. This directory must be created previously.                   | `"certs"`            |
 |`add_intermediates_to_bundle`| Add intermediate certificates into Bundle file instead of SVID file.                                        | `true`               |
 |`renew_signal`            | The signal that the process to be launched expects to reload the certificates. It is not supported on Windows. | `"SIGUSR1"`          |
 |`svid_file_name`          | File name to be used to store the X.509 SVID public certificate in PEM format.                                 | `"svid.pem"`         |
 |`svid_key_file_name`      | File name to be used to store the X.509 SVID private key and public certificate in PEM format.                 | `"svid_key.pem"`     |
 |`svid_bundle_file_name`   | File name to be used to store the X.509 SVID Bundle in PEM format.                                             | `"svid_bundle.pem"`  |

>  |`audience`                | JWT SVID audience.                                                                                             | `"example.org"`      |

Reviewer: ```suggestion
 |`jwt_audience`                | JWT SVID audience.                                                                                             | `"example.org"`      |
```

>  |`jwt_audience`            | JWT SVID audience.                                                                                             | `"example.org"`      |

Reviewer 2: ```suggestion
 |`jwt_audience`            | JWT SVID audience.                                                                                             | `"your-audience"`      |
```

>  |`jwt_file_name`           | File name to be used to store JWT SVID certificate in JSON format.                                             | `"jwt.json"`         |

Reviewer 3: JWTs aren't certificates

>  |`jwt_svid_file_name`      | File name to be used to store JWT SVID in JSON format.                                                         | `"jwt.json"`         |

Reviewer 3: This is no longer JSON format, right?

Reviewer: what format would it be in?

Author: Well, is a Json Web Token, don't know what would be the correct format. Any idea @keeganwitt ?

Reviewer 2: The JWT token is in a base64-encoded format.

We could change that documentation to:

```suggestion
 |`jwt_svid_file_name`      | File name to be used to store JWT SVID in Base64-encoded string.                                                         | `"jwt.json"`         |
```

Reviewer 3: Or if you wanted to have the names be similar to the X.509 SVID, it could be `svid.jwt` and `svid_bundle.json`.

>  |`jwt_audience`                | JWT SVID audience.                                                                                             | `"example.org"`      |
>  |`jwt_svid_file_name`      | File name to be used to store JWT SVID in JSON format.                                                         | `"jwt.json"`         |

Reviewer: ```suggestion
 |`jwt_audience`            | JWT SVID audience.                                                                                             | `"example.org"`      |
 |`jwt_svid_file_name`      | File name to be used to store JWT SVID in JSON format.                                                         | `"jwt.json"`         |
```

>  |`jwk_file_name`           | File name to be used to store JWT SVID Bundle in JSON format.                                                  | `"jwk.json"`         |

Reviewer 2: I would recommend renaming the configuration properties `jwt_file_name` and `jwk_file_name` to `jwt_svid_file_name` and `jwt_bundle_file_name` respectively. This change aims to prevent confusion by clearly distinguishing between the JWT SVID and the JWT bundle. The updated names make the intended use for each file more apparent and can help reduce the likelihood of misconfiguration.

svid_file_name = "svid.pem"
svid_key_file_name = "svid_key.pem"
svid_bundle_file_name = "svid_bundle.pem"

> audience = "example.org"

Reviewer: ```suggestion
jwt_audience = "example.org"
```

> jwt_audience = "example.org"

Reviewer 2: ```suggestion
jwt_audience = "your-audience"
```

> jwt_svid_file_name = "jwt.json"

Reviewer 2: ```suggestion
jwt_svid_file_name = "jwt.token"
```

svid_file_name = "svid.pem"
svid_key_file_name = "svid_key.pem"
svid_bundle_file_name = "svid_bundle.pem"

> audience = "example.org"

Reviewer: ```suggestion
jwt_audience = "example.org"
```

> jwt_svid_file_name = "jwt.json"

Reviewer 2: ```suggestion
jwt_svid_file_name = "jwt.token"
```
